code/game/objects/items/storage/smartpack.dm

```diff
@@ -201,19 +216,27 @@
 	to_chat(user, SPAN_INFO("The current charge reads [battery_charge]/[SMARTPACK_MAX_POWER_STORED]"))
 	update_icon(user)
 
+	var/filter_color = PROTECTIVE_FORM_COLOR
+	var/filter_size = EXOSKELETON_OFF_FILTER_SIZE
+	if(show_exoskeleton)
+		filter_size = EXOSKELETON_ON_FILTER_SIZE
+	var/filter_priority = 1
+	user.add_filter("synth_protective_form", filter_priority, list("type" = "outline", "color" = filter_color, "size" = filter_size))
```

Comment:
this isn't how you do named args
it's in the format proc(argname = argvalue, argname2 = argvalue2)

Reply:
wah